--- node_modules/sweetalert2/dist/sweetalert2.js	2022-11-23 07:20:19
+++ node_modules/sweetalert2/dist/sweetalert2.js	2022-11-23 07:24:00
@@ -398,15 +398,15 @@
       case 'select':
       case 'textarea':
       case 'file':
-        return popup.querySelector(`.${swalClasses.popup} > .${swalClasses[inputClass]}`);
+        return popup.querySelector(`.${swalClasses.popup} .${swalClasses[inputClass]}`);
       case 'checkbox':
-        return popup.querySelector(`.${swalClasses.popup} > .${swalClasses.checkbox} input`);
+        return popup.querySelector(`.${swalClasses.popup} .${swalClasses.checkbox} input`);
       case 'radio':
-        return popup.querySelector(`.${swalClasses.popup} > .${swalClasses.radio} input:checked`) || popup.querySelector(`.${swalClasses.popup} > .${swalClasses.radio} input:first-child`);
+        return popup.querySelector(`.${swalClasses.popup} .${swalClasses.radio} input:checked`) || popup.querySelector(`.${swalClasses.popup} .${swalClasses.radio} input:first-child`);
       case 'range':
-        return popup.querySelector(`.${swalClasses.popup} > .${swalClasses.range} input`);
+        return popup.querySelector(`.${swalClasses.popup} .${swalClasses.range} input`);
       default:
-        return popup.querySelector(`.${swalClasses.popup} > .${swalClasses.input}`);
+        return popup.querySelector(`.${swalClasses.popup} .${swalClasses.input}`);
     }
   };
