--- node_modules/sweetalert2/dist/sweetalert2.js	2022-11-23 07:20:19
+++ node_modules/sweetalert2/dist/sweetalert2.js	2022-11-23 07:24:00
@@ -535,15 +535,15 @@
       case 'select':
       case 'textarea':
       case 'file':
-        return popup.querySelector(".".concat(swalClasses.popup, " > .").concat(swalClasses[inputClass]));
+        return popup.querySelector(".".concat(swalClasses.popup, " .").concat(swalClasses[inputClass]));
       case 'checkbox':
-        return popup.querySelector(".".concat(swalClasses.popup, " > .").concat(swalClasses.checkbox, " input"));
+        return popup.querySelector(".".concat(swalClasses.popup, " .").concat(swalClasses.checkbox, " input"));
       case 'radio':
-        return popup.querySelector(".".concat(swalClasses.popup, " > .").concat(swalClasses.radio, " input:checked")) || popup.querySelector(".".concat(swalClasses.popup, " > .").concat(swalClasses.radio, " input:first-child"));
+        return popup.querySelector(".".concat(swalClasses.popup, " .").concat(swalClasses.radio, " input:checked")) || popup.querySelector(".".concat(swalClasses.popup, " .").concat(swalClasses.radio, " input:first-child"));
       case 'range':
-        return popup.querySelector(".".concat(swalClasses.popup, " > .").concat(swalClasses.range, " input"));
+        return popup.querySelector(".".concat(swalClasses.popup, " .").concat(swalClasses.range, " input"));
       default:
-        return popup.querySelector(".".concat(swalClasses.popup, " > .").concat(swalClasses.input));
+        return popup.querySelector(".".concat(swalClasses.popup, " .").concat(swalClasses.input));
     }
   };
